docs/api/structures/ipc-renderer-event.md

```diff
@@ -2,6 +2,7 @@
 
 * `sender` [IpcRenderer](../ipc-renderer.md) - The `IpcRenderer` instance that emitted the event originally
 * `senderId` Integer - The `webContents.id` that sent the message, you can call `event.sender.sendTo(event.senderId, ...)` to reply to the message, see [ipcRenderer.sendTo][ipc-renderer-sendto] for more information. This only applies to messages sent from a different renderer. Messages sent directly from the main process set `event.senderId` to `0`.
+* `isMainFrame` boolean (optional) - Whether the message sent via [ipcRenderer.sendTo][ipc-renderer-sendto] was sent by the main frame. This is relevant when `nodeIntegrationInSubFrames` is enabled in the originating `webContents`.
```

Comment:
Would exposing the origin additionally be helpful to distinguish across multiple frames ?

Reply:
it would, but that would add some additional overhead, I am planning to deprecate the API in #39091, this is just a minimal change to make it possible to ignore messages coming from non-main frames.